website/package.json

```diff
@@ -15,8 +15,11 @@
     "clsx": "^1.1.1",
     "docusaurus-plugin-remote-content": "^3.1.0",
     "dotenv": "^16.0.1",
+    "raw-loader": "^4.0.2",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "rehype-katex": "^4.0.0",
+    "remark-math": "^3.0.1"
```

Comment:
are these 3 new dependencies needed?

Reply:
Yes. When I tried to delete the 3 dependencies, the localserver returned an error: `Module not found: SyntaxError:...`

Reply:
you can remove them by running `npm uninstall <package-name>`

They should then be removed from both the `package-lock.json` and `package.json`

Reply:
After a quick search, it seems `"react-dom"` is needed for MDX and reusables.
Ran commands to remove the other two packages.